src/components/resource-selector/shared/resource-preview.less

```diff
@@ -11,6 +11,10 @@
     height: 100%;
   }
 
+  &.ResourcePreview--video {
+    height: auto;
+  }
```

Comment:
This is a bonus fix for this case:

**- before**
<img width="787" alt="Screenshot 2023-11-17 at 11 09 11" src="https://github.com/educandu/educandu/assets/8189923/fbd70d14-1d00-4cc0-8fe7-56e9c9ae4e40">


**- after**
<img width="773" alt="Screenshot 2023-11-17 at 11 11 54" src="https://github.com/educandu/educandu/assets/8189923/26f581db-1436-443e-8333-2cdf6987813f">